docs/conf.py

```diff
@@ -0,0 +1,77 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('..'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'lsst-panel'
+copyright = '2019, Quansight'
+author = 'Quansight'
+
+#need to add path so that function will work that will handle the case where sphinx will
+#ignore certain strings in the code.
+sys.path.insert(0, os.path.abspath('../../nbsite/'))
+from nbsite import nbbuild
```

Comment:
@ppwadhwa looks like you are using a new package called nbsite. Please add that the the environment.yml file.

Reply:
I added it. I had install pip as well and then pip install directly from github.  I'm not sure if there is a better way, but this is what I placed in the dependencies in the .yml file:
- pip:
      - git+https://github.com/pyviz-dev/nbsite.git

Reply:
ok so nbsite is available as a conda package on the pyviz channel (see https://anaconda.org/pyviz/nbsite). So we can add the pyviz channel to the top of the env file after the other two channels. Then you can just list nbsite as one of the conda packages.

What you did will work but we typically want to avoid pip package when a conda version is available on a reputable channel. In this case `pyviz` channel is maintained by the folks who develop the holoviews, datashader & panel packages.

Reply:
Got it.  I will make the change and try it out!